subprojects/tooling-api/src/integTest/groovy/org/gradle/integtests/tooling/ToolingApiResolveIntegrationTest.groovy

```diff
@@ -33,7 +33,13 @@ class ToolingApiResolveIntegrationTest extends AbstractIntegrationSpec {
                 ${mavenCentralRepository()}
             }
 
-            configurations { customConf }
+            configurations {
+                customConf {
+                    attributes {
+                        attribute(Category.CATEGORY_ATTRIBUTE, project.objects.named(Category, "library"))
```

Comment:
I wonder if we can avoid this somehow or make it nicer.

Reply:
I asked Louis, I will try @ljacomet 's suggestion today.

Reply:
Louis says this is currently the best workaround...